main.go

```diff
@@ -78,4 +102,6 @@ func init() {
 	flag.StringVar(&portBaseURL, "port-base-url", "https://api.getport.io", "Port base URL. Optional.")
 	portClientId = os.Getenv("PORT_CLIENT_ID")
 	portClientSecret = os.Getenv("PORT_CLIENT_SECRET")
+
+	eventListenerType = goutils.GetEnvOrDefault("EVENT_LISTENER__TYPE", "POLLING")
```

Comment:
I think we need to have a unified way to get configuration (now some of them from flags and some from env var), and to manage it in one central place and not looking for it in the code.